CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+# 3.17.2 (2021-02-09)
```

Comment:
I see 3.17.2 twice in this changelog :)

Reply:
ok I will fix and open a new PR me--

Reply:
Fixed with https://github.com/Katello/katello/pull/9173